src/components/Card/Card.tsx

```diff
@@ -5,8 +5,8 @@ interface CardProps {
   title: string
   image: string
   id: number
-  sentiment: number
-  updateUserSentiment: (userSentiment: number) => void
+  sentiment: number | any
+  updateUserSentiment: any
```

Comment:
I'm curious why this had to get changed from a more specific type to any. Same with line 8, why is number or any necessary? Would number or null work on the sentiment?

Reply:
Now that I'm taking a second look at this, I'm not actually sure how this was changed to any? I thought I made the change, but I don't know why I would have gotten rid of the type for updateUserSentiment. Maybe I made a mistake when resolving a merge conflict?

Regardless, it's back to normal now! Thanks for catching that.